docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/refget/about-refget.md

```diff
@@ -0,0 +1,375 @@
+
+In order to find an appropriate recording using our GraphQL service, we first need to find a `genome_id` to use in subsequent queries. We are using `"GCA_000001405.29`, which represents the human assembly GRCh38.p14 (hg38).
+
+```json
+{
+  genomes(by_assembly_accession_id: {assembly_accession_id: "GCA_000001405.29"}) {
+    genome_id
+  }
+}
+```
+
+This query will return a response similar to the following.
+
+```json
+{
+  "data": {
+    "genomes": [
+      {
+        "genome_id": "a7335667-93e7-11ec-a39d-005056b38ce3"
+      }
+    ]
+  }
+}
+```
+
+We can use the following GraphQL query to retrieve all known transcripts for the human BRCA2 gene using the previously retrieved `genome_id`.
+
+```json
+query {
+  genes(
+    by_symbol: {
+      symbol: "BRCA2",
+      genome_id: "a7335667-93e7-11ec-a39d-005056b38ce3"
+    }
+  ) {
+    stable_id
+    symbol
+    transcripts {
+      stable_id
+      symbol
+      metadata {
+        canonical {
+          value
+        }
+      }
+      product_generating_contexts {
+        product_type
+        cdna {
+          sequence {
+            checksum
+          }
+        }
+        cds {
+          sequence {
+            checksum
+          }
+        }
+        product {
+          stable_id
+          sequence {
+            checksum
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Each sequence product (cDNA, CDS and protein) refer to a checksum. The value of this field should be used verbatim with the refget service.
+
+```json
+{
+  "data": {
+    "genes": [
+      {
+        "stable_id": "ENSG00000139618.18",
+        "symbol": "BRCA2",
+        "transcripts": [
+          {
+            "stable_id": "ENST00000380152.8",
+            "symbol": "BRCA2-201",
+            "metadata": {
+              "canonical": {
+                "value": true
+              }
+            },
+            "product_generating_contexts": [
+              {
+                "product_type": "Protein",
+                "cdna": {
+                  "sequence": {
+                    "checksum": "2f8c687a656debd4c8b1b4d47747ba35"
+                  }
+                },
+                "cds": {
+                  "sequence": {
+                    "checksum": "c6b5e532b5aef696e987ca55a538ae9e"
+                  }
+                },
+                "product": {
+                  "stable_id": "ENSP00000369497.3",
+                  "sequence": {
+                    "checksum": "92addb948c6c652abc1dcecca05f26c0"
+                  }
+                }
+              }
+            ]
+          },
+          ....
+        ]
+      }
+    ]
+  }
+}
+```
+
+Each sequence can now be retrieved with the following URLs.
+
+- cDNA: <https://beta.ensembl.org/data/refget/sequence/2f8c687a656debd4c8b1b4d47747ba35>
+- CDS: <https://beta.ensembl.org/data/refget/sequence/c6b5e532b5aef696e987ca55a538ae9e>
+- Protein: <https://beta.ensembl.org/data/refget/sequence/92addb948c6c652abc1dcecca05f26c0>
```

Comment:
I think these links will show up on the help page styled as external, because we have a css rule that styles all links starting with http as external.

See discussion on slack: https://genomes-ebi.slack.com/archives/C027WCLBDH9/p1700589620674909

There are two options for how to deal with this.

One, at the level of the markdown files, is to write links as relative:

```
[https://beta.ensembl.org/data/refget/sequence/2f8c687a656debd4c8b1b4d47747ba35](/data/refget/sequence/2f8c687a656debd4c8b1b4d47747ba35)
```

another, at the level of css in ensembl-client, is to change the selector such that it ignores anything containing `"ensembl.org"`.

The first option places the responsibility on content authors, and thus is easy to forget. The second puts the responsibility on ensembl frontend; but would need a matching pattern that css can recognise.

Reply:
I guess the question is do we want them rendered as external or relative. If we want relative, I have no great feeling either way, then I am happy to change to relative